source/plugins/go/input/lib/cadvisor.go

```diff
@@ -0,0 +1,1219 @@
+		if cpuUsageNanoSecondsRate != nil {
+			metricDataItems = append(metricDataItems, cpuUsageNanoSecondsRate)
+		}
+
+		metricDataItems = append(metricDataItems, getNodeMetricItem(metricInfo, hostName, "memory", "workingSetBytes", MEMORY_WORKING_SET_BYTES, metricTime))
+	} else {
+		metricDataItems = append(metricDataItems, getContainerMemoryMetricItems(metricInfo, hostName, "workingSetBytes", MEMORY_WORKING_SET_BYTES, metricTime, operatingSystem, namespaceFilteringMode, namespaces)...)
+		metricDataItems = append(metricDataItems, getContainerStartTimeMetricItems(metricInfo, hostName, "restartTimeEpoch", metricTime, namespaceFilteringMode, namespaces)...)
+
+		if operatingSystem == "Linux" {
+			metricDataItems = append(metricDataItems, getContainerCpuMetricItems(metricInfo, hostName, "usageNanoCores", CPU_USAGE_NANO_CORES, metricTime, namespaceFilteringMode, namespaces)...)
+			metricDataItems = append(metricDataItems, getContainerMemoryMetricItems(metricInfo, hostName, "rssBytes", MEMORY_RSS_BYTES, metricTime, operatingSystem, namespaceFilteringMode, namespaces)...)
+			metricDataItems = append(metricDataItems, getNodeMetricItem(metricInfo, hostName, "memory", "rssBytes", MEMORY_RSS_BYTES, metricTime))
+		} else if operatingSystem == "Windows" {
+			containerCpuUsageNanoSecondsRate := getContainerCpuMetricItemRate(metricInfo, hostName, "usageCoreNanoSeconds", CPU_USAGE_NANO_CORES, metricTime, namespaceFilteringMode, namespaces)
+			if len(containerCpuUsageNanoSecondsRate) > 0 {
+				metricDataItems = append(metricDataItems, containerCpuUsageNanoSecondsRate...)
+			}
+		}
+
+		cpuUsageNanoSecondsRate := getNodeMetricItemRate(metricInfo, hostName, "cpu", "usageCoreNanoSeconds", CPU_USAGE_NANO_CORES, operatingSystem, metricTime)
+		if cpuUsageNanoSecondsRate != nil {
+			metricDataItems = append(metricDataItems, cpuUsageNanoSecondsRate)
+		}
+
+		metricDataItems = append(metricDataItems, getNodeMetricItem(metricInfo, hostName, "memory", "workingSetBytes", MEMORY_WORKING_SET_BYTES, metricTime))
+		metricDataItems = append(metricDataItems, getNodeLastRebootTimeMetric(metricInfo, hostName, "restartTimeEpoch", metricTime))
+		// Disabling networkRxRate and networkTxRate since we don't use it as of now.
+		// metricDataItems = append(metricDataItems, getNodeMetricItem(metricInfo, hostName, "network", "rxBytes", "networkRxBytes"))
+		// metricDataItems = append(metricDataItems, getNodeMetricItem(metricInfo, hostName, "network", "txBytes", "networkTxBytes"))
+		// networkRxRate := getNodeMetricItemRate(metricInfo, hostName, "network", "rxBytes", "networkRxBytesPerSec")
+		// if networkRxRate != nil && len(networkRxRate) > 0 {
+		// 	metricDataItems = append(metricDataItems, networkRxRate...)
+		// }
+		// networkTxRate := getNodeMetricItemRate(metricInfo, hostName, "network", "txBytes", "networkTxBytesPerSec")
+		// if networkTxRate != nil && len(networkTxRate) > 0 {
+		// 	metricDataItems = append(metricDataItems, networkTxRate...)
```

Comment:
looks like we can remove this code since this commented code in existing implementation, right?